packages/data-context/src/actions/DataEmitterActions.ts

```diff
@@ -35,6 +35,14 @@ abstract class DataEmitterEvents {
     this._emit('devChange')
   }
 
+  /**
+   * Emitted when cypress.config is re-executed and we'd like to
+   * either re-run a spec or update something in the App UI.
+   */
+  configChange () {
+    this._emit('configChange')
```

Comment:
Do we envision that this can eventually negate the need for some of the `toApp` and `toLaunchpad` calls elsewhere?

Reply:
Good question - for now I'm guessing we want to respond to any config changes in both. I can't think of any reason to selectively only update only `app` or only `launchpad` at this point.